app-emulation/firecracker-bin/firecracker-bin-0.21.1.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Secure and fast microVMs for serverless computing (static build)"
+HOMEPAGE="https://firecracker-microvm.github.io https://github.com/firecracker-microvm/firecracker"
+SRC_URI="
+	amd64? (
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/firecracker-v${PV}-x86_64
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/jailer-v${PV}-x86_64
+	)
+	arm64? (
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/firecracker-v${PV}-aarch64
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/jailer-v${PV}-aarch64
+	)"
+
+LICENSE="|| ( Apache-2.0 MIT Apache-2.0-with-LLVM-exceptions ) MPL-2.0"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+RESTRICT="test strip"
+
+RDEPEND="!app-emulation/firecracker
+	acct-group/kvm"
```

Comment:
How is acct-group/kvm used?

Reply:
Technically, it's not used at all. Firecracker needs KVM access, but the ebuild does not come with a user account to go into the `kvm` group. When using `jailer`, which is the recommended way of running `firecracker`, the point is moot anyway, since `jailer` sets up a chroot where `/dev/kvm` is accessible for the user running firecracker.
However, the dependency is in `app-emulation/firecracker`, so I believe this should be kept consistent between both ebuilds.